### PR TITLE
Initial pass at using brains for shared code

### PR DESCRIPTION
**Pull request by steam**
[#145627521](https://www.pivotaltracker.com/story/show/145627521)

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Nice, more deletions than insertions :red_circle:</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: package.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
     "cookie-parser": "1.4.3",
     "css-loader": "0.28.0",
     "dotenv": "4.0.0",
+    "ello-brains": "https://github.com/ello/brains.git",
```
Comment (jayzes) [inline package.json]: i think you might able to to shorthand this to `"ello-brains": "ello/brains",`

Comment (steam) [inline package.json]: It didn't work to modify this for some reason, @jayzes.

Comment (jayzes) [inline package.json]: Weird. ¯\_(ツ)_/¯ Oh well.

**Pull request merged**